elements/bulbs-video/components/revealed.js

```diff
@@ -247,7 +252,7 @@ export default class Revealed extends React.Component {
 
     player.setup(playerOptions);
 
-    GoogleAnalytics.init(player, videoMeta.gaPrefix);
+    GoogleAnalytics.init(player, videoMeta.sendAnalyticsEvent);
```

Comment:
Going to defer to @collin.  I still don't think we should pass by reference like this but not enough to say that has to be changed.